scripts/stack_tests.sh

```diff
@@ -7,6 +7,18 @@ GROUP=whoami
 INSTANCE_PREFIX=im-e2e
 INSTANCE_POSTFIX=$(tr -dc '[:lower:]' </dev/urandom | head -c 5; echo '')
 
+function cleanup_handler {
+  # shellcheck disable=SC2046
+  ./destroy.sh whoami $(./list.sh | jq -r '.[].Instances[]?.Name' | grep $INSTANCE_PREFIX | grep "$INSTANCE_POSTFIX")
```

Comment:
Just to make sure. I have not executed the bash fully in my head 😂 
Is there any chance we accidentally destroy the wrong instances?

Reply:
Not unless you create an instance with the same pre- and postfix

Reply:
To elaborate a bit more here, we have a fairly specific prefix. Even if we ignore the prefix and only focus on the postfix the following should be the case. The postfix consists of 5 lower case alphabetic characters, which gives us 26^5 (== 11881376) combinations. It should be reasonably safe.